File: JAVA TASK/assignment/BITTERKOLA/HourAndMinutes.java
import java.util.Scanner;
public class HourAndMinutes {

    public static void main(String[] args) {
    
    Scanner input = new Scanner(System.in);
    
    System.out.println("Enter hour : ");
    int hour = input.nextInt();

    System.out.println("Enter minutes: ");
    int minutes = input.nextInt();

    System.out.println("Enter hour ");
    int hourTwo = input.nextInt();

    System.out.println("enter minutes ");
    int minutesTwo = input.nextInt(); 
   
    System.out.println(hour + ":" + minutes);

    System.out.println(hourTwo + ":" + minutesTwo);
    
    int finalHour = hour + hourTwo;

    int finalMinute = minutes + minutesTwo;

    if (finalMinute >= 60) {
   finalMinute = finalMinute - 60;
    finalHour = finalHour + 1; 
  
}
  else if (finalHour <= 12) {
    finalHour = finalHour - 12; 
 } 
}
 }

File: JAVASCRIPT_TASK/HourAndMinutes.js
const prompt = require("prompt-sync") ();
const hour = number(prompt("Enter hour : "))

   const minutes = number(prompt("Enter minutes: ");

    const hourTwo = number(prompt("Enter hour ");

   const minutesTwo = number("enter minutes ");
   
  console.log(hour + ":" + minutes);

    console.log(hourTwo + ":" + minutesTwo);
    
    let finalHour = hour + hourTwo;

    let finalMinute = minutes + minutesTwo;

    if (finalMinute >= 60) {
   finalMinute = finalMinute - 60;
    finalHour = finalHour + 1; 
  
}
  else if (finalHour <= 12) {
    finalHour = finalHour - 12; 
 } 

